components/SignOut.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Head from 'next/head';
+
+const SignOut = () => {
+  console.log('SignOut being rendered')
+  return (
+    <div>
+      <Head>
```

Comment:
\<Head\> is used to insert code into the \<head\> of the page being rendered. It should not be used inside of a component.